### PR TITLE
feat: Providing docker version

### PR DESCRIPTION
**Pull request by pablosanches**
With this pull request we provide a docker version of skeleton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,15 @@ interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
...
-| documentation files                             | `docs/`                    |
-| web server files                                | `public/`                  |
-| other resource files                            | `resources/`               |
-| PHP source code                                 | `src/`                     |
-| test code                                       | `tests/`                   |
+|------------------------------------------------|----------------------------|
+| command-line executables                       | `bin/`                     |
+| configuration files                            | `config/`                  |
+| documentation files                            | `docs/`                    |
+| docker files                                   | `infra/`                   |
```
Comment (pmjones) [inline README.md]: The problem here (and below) is that we have no research to back up this recommendation. All the other recommendations derive from <https://github.com/php-pds/skeleton_research>.

**Review by pmjones — CHANGES_REQUESTED** [review]
Thanks for the contribution, but unless there is some research to back up the recommendation, it's not appropriate for this publication.

File: README.md (hunk 1 above)
Comment (pablosanches) [reply README.md]: Thanks for the feedback, it's an honor to be able to contribute. The work done in the research repository is incredible. In the research repository, I found a zend framework skeleton that provides docker support and the "infra" path that I added does not actually qualify. I made the requested adjustments, I hope I got it right this time :)

Thank you very much for your attention!

**Comment by glensc** [comment]
yeah, the pr title/body says docker, but actually there's dockerfile, makefile, and docker-compose

**Pull request closed without merging**